docs/user-guide/rug/generators.md

```diff
@@ -67,48 +156,32 @@ subdirectories.
 !!! note ""
     Because all of the Atomist files are hidden under the `.atomist`
     directory, our generator project is still a fully functioning,
-    perfectly valid MkDocs project.
+    perfectly valid Spring Boot project.
 
 Let's take a close look at the Rug generator script.
 
 ## A Basic Generator Script
 
 The generator script's `#!typescript populate` method is invoked after
 the model project's files have been copied to the target project.  A
-possible generator script for the MkDocs generated discussed above
-could look like this:
+possible generator script for the project discussed above could look like this:
 
 ```typescript linenums="1"
-import { PopulateProject } from "@atomist/rug/operations/ProjectGenerator";
-import { Generator, Tags } from "@atomist/rug/operations/Decorators";
-import { Pattern } from "@atomist/rug/operations/RugOperations";
-import { Project } from "@atomist/rug/model/Project";
-
-@Generator("NewMkDocsDocumentationProject", "creates a MkDocs project")
-@Tags("markdown", "documentation", "mkdocs")
-class NewMkDocsDocumentationProject implements PopulateProject {
+import { PopulateProject } from '@atomist/rug/operations/ProjectGenerator'
```

Comment:
We should consistently use semicolons at the end of lines. Apparently its idiomatic in TypeScript.